frontend/packages/integration-tests-cypress/support/login.ts

```diff
@@ -69,7 +69,13 @@ Cypress.Commands.add(
           }
         });
       },
-      { cacheAcrossSpecs: true },
+      {
+        cacheAcrossSpecs: true,
+        validate() {
+          // If unable to visit the home page, login has failed, and the session will be re-created.
+          cy.visit('/');
```

Comment:
Is this enough to validate that we are "able to visit the home page"? Do we need to assert something, like the existence of some DOM element?

Reply:
This fixed the problem. Any error thrown in the block leads to the session considered invalid. It made sense to me to just try to visit the landing page, where any attempt at that without logging in would lead to a redirect back to logging in.

Reply:
I'm going to add these changes to https://github.com/openshift/console/pull/13242 so we can improve the pass rate.